src/components/navbar/index.tsx

```diff
@@ -4,16 +4,16 @@ import PropTypes from 'prop-types';
 
 const Navbar = ({ page }) => {
   const navBarContent = (title, refUrl) => {
-    const navBarText = title === 'Tasks' ? 'Index' : title;
+    // const navBarText = title === 'Tasks' ? 'Index' : title;
```

Comment:
@sumitd94 I can't figure out what this line of code does, can you please explain>
I commented it to get the Tasks title on page.

Reply:
As Discussed

Reply:
Done with requested changes, please review.